frontend/src/routes/index.svelte

```diff
@@ -139,14 +139,28 @@
                 autofocus
             />
         </div>
-        <div class="grid grid-cols-1 sm:grid-cols-2 gap-2 py-1 px-4">
-            <MultiSelect --sms-options-bg="var(--my-css-var, #404454)"
+        <div class="sm:grid sm:grid-cols-2 sm:gap-2 px-4">
+            <MultiSelect
+                --sms-options-bg="#2a2e37"
+                --sms-border="1pt solid #6B7280"
+                --sms-focus-border="2pt solid #6B7280"
+                --sms-active-color="#2a2e37"
+                --sms-remove-x-hover-focus-color="#6B7280"
+                --sms-li-selected-bg="#3d4451"
+                --sms-li-active-bg="#059669"
```

Comment:
Could be made into a component #26

Reply:
This is quite hard since `MultiSelect` calls `debounceInput()` i think we have to rethink this and do this another time? But i feel like we need to get this PR in asap.

Reply:
I see, we can do it in another PR, not a major change :DD